inst/@sym/imageset.m

```diff
@@ -0,0 +1,82 @@
+%% Copyright (C) 2017 Lagu
+%%
+%% This file is part of OctSymPy.
+%%
+%% OctSymPy is free software; you can redistribute it and/or modify
+%% it under the terms of the GNU General Public License as published
+%% by the Free Software Foundation; either version 3 of the License,
+%% or (at your option) any later version.
+%%
+%% This software is distributed in the hope that it will be useful,
+%% but WITHOUT ANY WARRANTY; without even the implied warranty
+%% of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See
+%% the GNU General Public License for more details.
+%%
+%% You should have received a copy of the GNU General Public
+%% License along with this software; see the file COPYING.
+%% If not, see <http://www.gnu.org/licenses/>.
+
+%% -*- texinfo -*-
+%% @documentencoding UTF-8
+%% @defmethod @@sym imageset (@var{f}, @var{d})
+%% @defmethodx @@sym imageset (@var{f}, @var{x}, @var{d})
+%% Image of a set under a mathematical function.
+%% The imageset is created in the function @var{f} in the variable
+%% @var{x} on the domain @var{d}, if @var{x} don't is passed will
```

Comment:
`if @var{x} is not passed, it will be detected using symvar, @pxref{@@sym/symvar}.`

Or something like that.